.noseids

```diff
@@ -0,0 +1,100 @@
+(dp1
+S'failed'
+p2
+(lp3
+S'1'
+aS'2'
+aS'3'
+aS'4'
+aS'5'
+aS'6'
+aS'7'
+aS'8'
+aS'11'
+p4
+asS'source_names'
+p5
+(lp6
+S'/home/rossie/Documents/bucket_list/app/models/tests/user_tests.py'
+p7
+aS'app/models/tests/user_tests.py'
+p8
+aS'/home/rossie/Documents/bucket_list/app/models/tests/bucketlist_tests.py'
+p9
+aS'app/models/tests/user_tests.py:Testuser.test_create_bucket_item'
+p10
+aS'.'
+asS'ids'
+p11
+(dp12
+I1
+(S'/home/rossie/Documents/bucket_list/app/models/tests/user_tests.py'
+S'app.models.tests.user_tests'
+Ntp13
+sI2
+(S'/home/rossie/Documents/bucket_list/app/models/tests/user_tests.py'
+S'app.models.tests.user_tests'
+p14
+S'test_user_existence'
+tp15
+sI3
+(S'/home/rossie/Documents/bucket_list/app/models/tests/user_tests.py'
+g14
+S'test_register_user'
+tp16
+sI4
+(S'/home/rossie/Documents/bucket_list/app/models/tests/user_tests.py'
+g14
+S'test_create_bucketlist'
+tp17
+sI5
+(S'/home/rossie/Documents/bucket_list/app/models/tests/user_tests.py'
+g14
+S'test_create_bucket_item'
+tp18
+sI6
+(S'/home/rossie/Documents/bucket_list/app/models/tests/bucketlist_tests.py'
+S'app.models.tests.bucketlist_tests'
+S'test_create_item'
+tp19
+sI7
+(S'/home/rossie/Documents/bucket_list/app/models/tests/user_tests.py'
+S'app.models.tests.user_tests'
+S'Testuser.test_create_bucket_item'
+tp20
+sI8
+(S'/home/rossie/Documents/bucket_list/app'
+S'app'
+Ntp21
+sI9
+(S'/home/rossie/Documents/bucket_list/app/models/tests/bucketlist_tests.py'
+S'app.models.tests.bucketlist_tests'
+S'Testbucketlist.test_create_item'
+tp22
+sI10
+(S'/home/rossie/Documents/bucket_list/app/models/tests/user_tests.py'
+S'app.models.tests.user_tests'
+p23
+S'Testuser.test_create_bucketlist'
+tp24
+sI11
+(S'/home/rossie/Documents/bucket_list/app/models/tests/user_tests.py'
+g23
+S'Testuser.test_del_bucketlist'
+tp25
+sI12
+(S'/home/rossie/Documents/bucket_list/app/models/tests/user_tests.py'
+g23
+S'Testuser.test_get_username'
+tp26
+sI13
+(S'/home/rossie/Documents/bucket_list/app/models/tests/user_tests.py'
+g23
+S'Testuser.test_register_user'
+tp27
+sI14
+(S'/home/rossie/Documents/bucket_list/app/models/tests/user_tests.py'
+g23
+S'Testuser.test_user_existence'
+tp28
+ss.
```

Comment:
I have tried.
Even included it in the .gitignore file but it is not getting ignored